pubspec.yaml

```diff
@@ -46,6 +47,7 @@ dependencies:
 
 dependency_overrides:
   dartx: ^0.4.1
+  path: 1.7.0
```

Comment:
What is this override for?

Reply:
[provider_architecture_template] flutter pub get
Running "flutter pub get" in provider_architecture_template...     
Because every version of flutter_test from sdk depends on path 1.7.0 and hive_flutter 0.3.0+2 depends on path >=1.6.0 <1.7.0, flutter_test from sdk is incompatible with hive_flutter 0.3.0+2.

And because no versions of hive_flutter match >0.3.0+2 <0.4.0, flutter_test from sdk is incompatible with hive_flutter ^0.3.0+2.

So, because provider_start depends on both hive_flutter ^0.3.0+2 and flutter_test any from sdk, version solving failed.
pub get failed (1; So, because provider_start depends on both hive_flutter ^0.3.0+2 and flutter_test any from sdk, version solving failed.)
exit code 1

Reply:
I fixed it temporarily with the dartx override. Can you try removing the path and seeing if just the dartx override works for you? Might need to run `flutter run upgrade`

Reply:
I am on beta channel, and removing path does not work. However, removing the darts override works when the path override is present

Reply:
Alright if the path works without the dartx. Can you test on stable and then i'll merge if all good

Reply:
just pulled master channel. On master it needs the dartx override, and not the path one, but on beta it needs path not dartx.

Reply:
Alright i can change it later if hive fixes its generator dependencies